### PR TITLE
Library component

### PR DESCRIPTION
**Pull request by oliferwolox**
## Summary

Add Library component for listing books.

## Screenshots

Screenshot_1586443578

## Trello Card

https://trello.com/c/9LRY5mbd/9-lista-de-libros

### CODE REVIEW

File: src/app/components/Library/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+import styles from './styles';
+
+function Library({ books }) {
+  const renderBook = useCallback(
+    ({ item }) => <Book title={item.title} author={item.author} image={item.imageUrl} />,
+    []
+  );
+
+  const extractBookId = useCallback((item) => item.id.toString(), []);
```
Comment (IgnacioAbadie) [inline src/app/components/Library/index.js:16]: just a comment. Usually we call this like the flatlist prop. `keyExtractor`

File: src/propTypes/book.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { shape, string, number } from 'prop-types';
+
+export const bookProps = shape({
```
Comment (IgnacioAbadie) [inline src/propTypes/book.js:3]: I like your approach but i prefer think it like the `Model` in this case `BookModel` into `models` folder.
What do you think about? @marianozicavo @oliferwolox

File: src/app/components/Library/styles.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+  library: {
+    flex: 1,
+    marginTop: 20,
```
Comment (marianozicavo) [inline src/app/components/Library/styles.js:6]: Use `marginVertical` for this

File: src/app/components/Library/styles.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
```
Comment (marianozicavo) [inline src/app/components/Library/styles.js:3]: We always put the `export default` at the end

File: src/propTypes/book.js (hunk 2 above)
Comment (marianozicavo) [reply src/propTypes/book.js:3]: I think I like it better this way, because we only  `shape()` once

**Pull request merged**